### PR TITLE
Crypto🔗Genesis  / terravaloper1j73n84drqm6q5wttadr9meg6jcldlkd5rq0u5l

### PR DESCRIPTION
**Pull request by Crypt0Genesis**
New Validator PR
Crypto🔗Genesis
terravaloper1j73n84drqm6q5wttadr9meg6jcldlkd5rq0u5l

### CODE REVIEW

File: validators/terravaloper1j73n84drqm6q5wttadr9meg6jcldlkd5rq0u5l/profile.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "contact": {
+    "email": "contact@cryptogenesis.info",
+    "telegram": "@crypt0genesis",
+    "twitter": "@Crypto0Genesis",
+    "discord": "Crypto🔗Genesis"
```
Comment (terran6) [inline validators/terravaloper1j73n84drqm6q5wttadr9meg6jcldlkd5rq0u5l/profile.json:6]: Missing a comma on the Discord line, causing the file to not have proper JSON formatting.

**Review by terran6 — CHANGES_REQUESTED** [review]

File: validators/terravaloper1j73n84drqm6q5wttadr9meg6jcldlkd5rq0u5l/profile.json (hunk 1 above)
Comment (Crypt0Genesis) [reply validators/terravaloper1j73n84drqm6q5wttadr9meg6jcldlkd5rq0u5l/profile.json:6]: Hi, let me check and fixed it.

Comment (Crypt0Genesis) [reply validators/terravaloper1j73n84drqm6q5wttadr9meg6jcldlkd5rq0u5l/profile.json]: > Missing a comma on the Discord line, causing the file to not have proper JSON formatting.

Fixed and I have updated the repository. Please check and let me know. Thank you!!

Comment (Crypt0Genesis) [reply validators/terravaloper1j73n84drqm6q5wttadr9meg6jcldlkd5rq0u5l/profile.json]: Do I have to pull new request?

**Review by Crypt0Genesis — COMMENTED** [review]
Issue fixed, please review. Thank you

**Review by terran6 — APPROVED** [review]
🚀

**Pull request merged**